src/WinRPM.jl

```diff
@@ -321,11 +321,7 @@ struct RPMVersionNumber
 end
 Base.convert(::Type{RPMVersionNumber}, s::AbstractString) = RPMVersionNumber(s)
 Base.:(<)(a::RPMVersionNumber, b::RPMVersionNumber) = false
-Base.:(==)(a::RPMVersionNumber, b::RPMVersionNumber) = true
```

Comment:
I don't think you can remove this definition. Have you tested it? OTC, is `<` really needed?

(I must say these constant definitions are really weird...)

Reply:
I did test it on all operations except for install because it is still failing due to a follow-on error.  I can put it back for now until I can test end-to-end without any issues.

< will be needed for backwards compatibility since maximum() calls < and not isless in .6

Reply:
I also agree they are really weird.  I think because there is a lot of repetition in the package list it is just trying to grab the first or last entry which represents the oldest or newest and really isn't doing any number based comparison.

Reply:
this package is undertested, difficult to test, but heavily depended on, so please minimize the changes you make to it to be as low risk as possible. particularly if you are fixing 0.7-only issues, don't mix concerns and risk breaking 0.6.

Reply:
I agree with @tkelman  I rolled my changes back to just adding isless to get .7 to work.